### PR TITLE
Simple test cases covering functionality and basic loads

### PR DESCRIPTION
**Pull request by pct960**
Test cases cover
* RTMPS
* RTMPS + HLS
* Recording
* Different video formats
* HLS
* HD video tests
* Simple loads

### CODE REVIEW

**Comment by pct960** [comment]
@Gtgstg Feel free to add details about more tests if I've missed out on anything.

File: setup/video-server.conf [hunk 1]
```diff
@@ -14,4 +14,4 @@ SSL_CERT_PASSWORD=password
 
 #Postgres options
 POSTGRES_USER=postgres
-POSTGRES_PASSWORD=
+POSTGRES_PASSWORD=9e7a242bd9a77f95f784c4525ee13af0a9fa0061
```
Comment (pct960) [inline setup/video-server.conf:17]: Postgres password need not be committed.

File: test/config.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+RESOURCE_ID = {
+    "test-resource-1": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource-1",
+    "test-resource-2": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource-2",
+    "test-resource-3": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource-3",
+    "test-resource-4": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource-4",
+    "public": "rbccps.org/e096b3abef24b99383d9bd28e9b8c89cfd50be0b/example.com/test-category/test-resource.public"
+}
+
+YOUTUBE_URL = ['https://www.youtube.com/watch?v=iDO9J_3OVJ0',
```
Comment (pct960) [inline test/config.py:11]: Make this also a ``dict`` with keys?

File: test/config.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+RTMP = 'rtmp'
+PUSH_VALID = 'push_valid'
+PUSH_INVALID = 'push_invalid'
+PLAY_VALID = 'play_valid'
+PLAY_INVALID = 'play_invalid'
+FRAME_RATE = 'frame_rate'
+
+RECORD_SRC_DIR = '../nginx/record/'
+
+ACL_SET_ENDPOINT = 'https://localhost:8443/auth/v1/acl/set'
```
Comment (pct960) [inline test/config.py:34]: This should use the ``AUTH_SERVER`` environment variable instead of using ``localhost:8443``

File: test/start_tests [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        headers={"content-type": "application/json"}
+    )
+
+    logging.info("Requesting access tokens")
+    resp = requests.post(
+        url=conf.REQUEST_TOKEN_ENDPOINT,
+        verify=False,
+        cert=conf.REQUEST_TOKEN_CREDENTIALS,
+        data=conf.REQUEST_TOKEN_BODY,
+        headers={"content-type": "application/json", "Host": "auth.local"}
```
Comment (pct960) [inline test/start_tests:37]: ``auth.local`` should not be hardcoded

**Comment by Gtgstg** [comment]
> @Gtgstg Feel free to add details about more tests if I've missed out on anything.

**Pull request merged**